lang/cs/Org.Apache.REEF.Common/Services/ServiceConfiguration.cs

```diff
@@ -62,6 +72,17 @@ public static ConfigurationModule ConfigurationModule
             }
         }
 
+        public static string WrapServiceConfigAsString(IConfiguration serviceConfiguration)
```

Comment:
Can we make this non-public? Also, please add documentation such that we remember why we did the whole wrapping exercise.